### DISCUSSION TITLE
Sensor charge en discharge power

### DISCUSSION THREAD

**Discussion by vdvmichel** [TypQxQ/Sigenergy-Home-Assistant-Integration#6] [Ideas]
Thanks for the integration.
Is it possible to have more sensors?
For example in huawei I can say battery charge power 1kw.
My installation is over-sized... now the battery charge in the morning and it is full by noon... if I can set it to charge at 1 kW in the morning then it will not be full in the afternoon and I can adjust the over-sizing there let the battery charge... now I have lost this energy due to the capping of the inverter...
I have also in other home installation of huawei and I have with that integration al lot of sensors that I can change.
Als is het possible to not charge ac?

**Comment by TypQxQ** [comment]
This are all the available sensors and options exposed by Sigenergy.
Are you looking for "Sigen ESS max charging limit" maybe?
If you only want to limit charging to 1kW, set it to 1.

Regards,
Andrei

**Comment by TypQxQ** [comment]
Closing now because of inactivity.
